src/keri/vdr/verifying.py

```diff
@@ -408,8 +408,9 @@ def verifyChain(self, nodeSaid, op):
             if iss is None:
                 return None
 
-            if op == 'I2I' and nodeSaid not in [i.qb64 for i in iss]:
-                return None
+            if op == 'I2I':
+                if issuer != creder.subject['i'] or nodeSaid not in [i.qb64 for i in iss]:
```

Comment:
I'm confused with the check for nodeSaid in iss wasn't enough.  That should be redundant to the check of issuer against creder.subject['i'].  If the subjs database is an index for look ups and not required to check for valid credential.  So just issuer != creder.subject['i'] is needed.

Reply:
Good call, I thought subjs was validating more than it was. I've made the change, and tested against my acdc graphs and things look okay.